tests/func/test_data_cloud.py

```diff
@@ -290,6 +291,20 @@ def _get_cloud_class(self):
         return RemoteHDFS
 
 
+@pytest.mark.usefixtures("http_server")
+class TestRemoteHTTP(HTTP, TestDataCloudBase):
+    @pytest.fixture(autouse=True)
+    def setup_method_fixture(self, request, http_server):
+        self.http_server = http_server
+        self.method_name = request.function.__name__
+
+    def get_url(self):
```

Comment:
I think DS has a valid point here, the original method is static, shouldn't we make all of them non-static now, that it is required by this particular change?

Reply:
For what it's worth, I wrote it like this because it was done the same way in `TestRemoteSSHMocked` (which overrides static `SSHMocked.get_url()` from `tests/remotes.py`).

Reply:
Well, it was only a question, its not severe, because any potential problems should be detected on the build stage. For me, it can stay as is.